jeongjaein/ElevenStreetSample/app/src/main/java/com/exam/elevenstreet/data/repository/ProductRepository.kt

```diff
@@ -0,0 +1,29 @@
+package com.exam.elevenstreet.data.repository
+
+import com.exam.elevenstreet.data.CallBack
+import com.exam.elevenstreet.data.source.local.ProductLocalDataSource
+import com.example.elevenstreet.ProductResponse
+
+class ProductRepository(
+    val productLocalDataSource: ProductLocalDataSource
```

Comment:
```suggestion
    private val productLocalDataSource: ProductLocalDataSource
```
외부에서 접근하지 못하게 접근 한정자를 붙이면 좋을 것 같아요,

Reply:
반영하겠습니다